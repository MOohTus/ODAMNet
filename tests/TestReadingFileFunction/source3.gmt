REAC:R-HSA-109704	PI3K Cascade	GAB2	PIK3CB	FGF10	FGF22	FGF4	PIK3C3	FGF20	FLT3LG	TRIB3	FGF9	AKT2	PIK3R2	GAB1	FGF6	FGF1	FGF23	PIK3CA	FLT3	KLB	FGF2	FGF7	PDPK1	PIK3R1	PDE3B	FGF18	THEM4	FGFR4	FGF19	FRS2	IRS1	PTPN11	IRS2	FGF3	PIK3R4	FGF16	TLR9
REAC:R-HSA-110056	MAPK3 (ERK1) activation	JAK2	MAPK3	TYK2	IL6	IL6R	JAK1	MAP2K1	CDK1	PTPN11
REAC:R-HSA-110312	Translesion synthesis by REV1	REV3L	RFC1	RFC2	RPA3	RFC5	MAD2L2	RPA2	RPA1	PCNA	RFC3	REV1	RPS27A	UBC	RFC4	UBB	UBA52
REAC:R-HSA-110313	Translesion synthesis by Y family DNA polymerases bypasses lesions on DNA template	REV3L	SPRTN	RFC1	RFC2	POLD1	UFD1	POLD3	POLE2	POLI	USP10	RPA3	POLD2	RFC5	POLE4	MAD2L2	RPA2	TRIM25	POLK	RPA1	PCNA	RFC3	REV1	RPS27A	POLE3	UBC	USP43	UBE2L6	RCHY1	RFC4	VCP	PCLAF	UBB	POLH	POLD4	POLE	UBA7	NPLOC4	ISG15	UBA52
