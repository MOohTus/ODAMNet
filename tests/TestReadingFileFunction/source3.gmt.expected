AKT2
CDK1
FGF1
FGF10
FGF16
FGF18
FGF19
FGF2
FGF20
FGF22
FGF23
FGF3
FGF4
FGF6
FGF7
FGF9
FGFR4
FLT3
FLT3LG
FRS2
GAB1
GAB2
IL6
IL6R
IRS1
IRS2
ISG15
JAK1
JAK2
KLB
MAD2L2
MAP2K1
MAPK3
NPLOC4
PCLAF
PCNA
PDE3B
PDPK1
PIK3C3
PIK3CA
PIK3CB
PIK3R1
PIK3R2
PIK3R4
POLD1
POLD2
POLD3
POLD4
POLE
POLE2
POLE3
POLE4
POLH
POLI
POLK
PTPN11
RCHY1
REV1
REV3L
RFC1
RFC2
RFC3
RFC4
RFC5
RPA1
RPA2
RPA3
RPS27A
SPRTN
THEM4
TLR9
TRIB3
TRIM25
TYK2
UBA52
UBA7
UBB
UBC
UBE2L6
UFD1
USP10
USP43
VCP
